### DISCUSSION TITLE
Error updating changedetection.io

### DISCUSSION THREAD

**Discussion by Sciroppato** [tteck/Proxmox#3438] [Help Needed] [answered]
Hello!

On an updated debian 12, if i try to launch "update" i get this output:

Updating Change Detection LXC
[ERROR] in line 63: exit code 0: while executing command pip3 install changedetection.io --upgrade &> /dev/null

What can I do?

Thank you very much!

**Comment by tteck** [comment, marked as answer]
Search closed issues: https://github.com/tteck/Proxmox/issues/3400
